dips/dip-158.md

```diff
@@ -0,0 +1,805 @@
+---
+dip: 158
+title: P2M (eCommerce) Payments Off-Chain Protocol
+authors: Yinon Zohar, Daniel Prinz
+status: Draft
+type: Standard
+created: 03/15/2021
+---
+
+# Summary
+This DIP is an extension of the Off-Chain protocol for enabling eCommerce P2M payments (Peer To Merchant) of the following types:
+* Charge
+* Auth / Capture
+
+This DIP defines the type of eCommerce P2M payments, corresponding requests and the payment structures. 
+ 
+This DIP makes the Auth/Capture section in [DIP-8](https://dip.diem.com/dip-8/#authcapture) redundant. The auth/capture section in DIP-8 is to be disregarded and will be removed in a future version of DIP-8. 
```

Comment:
let's delete this line, personally, I would have preferred if we made this DIP-8 and made DIP-158 about recurring payments. Perhaps we should just kill DIP-8 altogether :).

Reply:
DIP-8 is needed for funds pre-approval.
At the moment DIP-8 contains a section about auth/capture that should be ignored as it is accurately specified in DIP-158.
As stated, once DIP-8 is changed to include only pre-approval specification I will delete this comment from DIP-158.